eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -59,6 +64,12 @@
     },
     {
       "StaticConfigs": {
+        "Win2019": {
+          "OSVmImage": "windows-2019",
+          "Pool": "Azure Pipelines",
+          "RunProxyTests": true,
+          "CMAKE_GENERATOR": "Visual Studio 16 2019"
+        },
```

Comment:
This part of the matrix was specifically intended to be run in Windows 2022 because of the proxy testing. It might work in 2019 but we moved it to 2022 because previous attempts to get WSL working in Windows 2019 failed and only succeeded on Windows 2022.